### ISSUE TITLE
display-hidpi-scale support on Windows 10

### ISSUE THREAD

**Issue by stax76** [mpv-player/mpv#7547]
https://github.com/mpv-player/mpv/labels/meta%3Afeature-request

### Expected behavior of the wanted feature

When I show the property [display-hidpi-scale](https://mpv.io/manual/master/#command-interface-display-hidpi-scale) on the OSD console I expect a value but it says unavailable. This is on Windows 10 with different vo settings tried and none returned a value, primary active screen uses 288 DPI. It causes the OSD console font defaults to be unusable, fortunately the OSD console allows to define a font size and scale value.

### Alternative behavior of the wanted feature

Instead of returning the DPI scale of the current screen it would be OK to return the DPI scale of the primary screen as return by following code:

```
    HDC screen = GetDC(NULL);
    g_DpiScale = GetDeviceCaps(screen, LOGPIXELSX) / (float)96;
    ReleaseDC(NULL, screen);
```

### Log file

https://pastebin.com/fy4Xtj2p

The CLI service isn't working here with Windows Terminal and PowerShell or I'm doing it wrong.

```
Desktop> curl.exe -F'file=C:\Users\frank\Desktop\mpv.log' https://0x0.st
Segmentation fault
Desktop> curl.exe -F'C:\Users\frank\Desktop\mpv.log' https://0x0.st
Warning: Illegally formatted input field!
curl: option -FC:\Users\frank\Desktop\mpv.log: is badly used here
curl: try 'curl --help' for more information
```

**Comment by stax76** [comment]
Default font size for the OSD console is OK in the last build.

**Comment by avih** [comment]
> Default font size for the OSD console is OK in the last build.

@stax76 You mean it got fixed? can you give the mpv hashes of the lask known bad version and first known good version?

**Comment by stax76** [comment]
Yes it's fixed, it works since the build from today I think. Thanks for fixing.

**Comment by avih** [comment]
So the issue was that `display-hidpi-scale` was unavailable? yeah, it was added three weeks ago - 6193f723b0aa234bfb5c869d9e2af67b0df0dfef .

**Comment by avih** [comment]
> The CLI service isn't working here with Windows Terminal and PowerShell or I'm doing it wrong.
> 
> ```
> Desktop> curl.exe -F'file=C:\Users\frank\Desktop\mpv.log' https://0x0.st
> Segmentation fault
> Desktop> curl.exe -F'C:\Users\frank\Desktop\mpv.log' https://0x0.st
> Warning: Illegally formatted input field!
> curl: option -FC:\Users\frank\Desktop\mpv.log: is badly used here
> curl: try 'curl --help' for more information
> ```

You're missing `@`, and depending on your shell, you might want to use double quotes for the whole argument, like so:
```sh
curl.exe "-Ffile=@C:\Users\frank\Desktop\mpv.log" https://0x0.st
```

**Comment by avih** [comment]
> Yes it's fixed, it works since the build from today I think. Thanks for fixing.

FWIW, It should be fixed in the last 3 builds:
```
mpv-x86_64-20210905-git-8aef22e.7z 	2021-09-05
mpv-x86_64-20210829-git-dffb94f.7z 	2021-08-29
mpv-x86_64-20210815-git-0c9e1e3.7z 	2021-08-15
```
While not yet fixed in `mpv-x86_64-20210801-git-416668d.7z 	2021-08-01`
